tmuxinator/aerostack2.yaml

```diff
@@ -0,0 +1,76 @@
+<%
+# Input parameters
+drone_namespace        = @settings["drone_namespace"]
+simulation_config_file = @settings["simulation_config_file"]
+base_launch            = @settings["base_launch"] == "true"
+behavior_tree          = @settings["behavior_tree"] == "true"
+%>
+name: <%= drone_namespace %>
+attach: false
+root: ./
+startup_window: mission_monitoring
+windows:
+  # Platform
+  - platform:
+      layout: tiled
+      panes:
+      # Platform
+      <%- if base_launch %>
+        - ros2 launch as2_gazebo_assets launch_simulation.py
+          use_sim_time:=true
+          simulation_config_file:=<%= simulation_config_file %>
+      <%- end %>
+        - ros2 launch as2_platform_gazebo platform_gazebo_launch.py
+            namespace:=<%= drone_namespace %>
+            platform_config_file:=config/config.yaml
+            simulation_config_file:=<%= simulation_config_file %>
+
+  # Basic Robotics Functions
+  - basics_robotics_functions:
+      layout:
+      panes:
+        # State Estimation
+        - ros2 launch as2_state_estimator state_estimator_launch.py
+            namespace:=<%= drone_namespace %>
+            config_file:=config/config.yaml
+        # Motion Controller
+        - ros2 launch as2_motion_controller controller_launch.py
+            namespace:=<%= drone_namespace %>
+            config_file:=config/config.yaml
+            plugin_name:=pid_speed_controller
+            plugin_config_file:=config/pid_speed_controller.yaml
+
+  # Behaviors
+  - behaviors:
+      layout:
+      panes:
+        # Motion Behaviors
+        - ros2 launch as2_behaviors_motion motion_behaviors_launch.py
+            namespace:=<%= drone_namespace %>
+            config_file:=config/config.yaml
+        # Trajectory Generation Behavior
+        - ros2 launch as2_behaviors_trajectory_generation generate_polynomial_trajectory_behavior_launch.py
+            namespace:=<%= drone_namespace %>
+            config_file:=config/config.yaml
+        # Point Gimbal Behavior
+        - ros2 launch as2_behaviors_perception point_gimbal_behavior.launch.py
+            namespace:=<%= drone_namespace %>
+            config_file:=config/config.yaml
+
+  # Mission execution
+  - mission_execution:
+      layout:
+      panes:
+        <%- if behavior_tree %>
+        - ros2 launch as2_behavior_tree behavior_trees.launch.py
```

Comment:
Why not to launch it always as we are doing with traj_generation and point_gimbal behaviors?

Reply:
It could be done. Do you think it's better?

Reply:
Not sure, but all missions would be available with the non-options launch and we would have less launch options which is more readable. I believe drawbacks are minimal